static/css/v3/heros.css

```
@@ -69,6 +69,114 @@
+  .hero--release .hero-fg__img {
+    object-fit: cover;
+  }
+
+  .hero--release.hero--with-background-image,
+  .dark .hero--release.hero--with-background-image,
+  [data-theme="dark"] .hero--release.hero--with-background-image {
+    background-image: none;
+    background-color: #5AB8EC;
+  }
+}
+}
```

Stylelint (17.14.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)